docs/user-guide/how_to_use_gpu_sharing.md

```diff
@@ -34,9 +34,10 @@ data:
       - name: conformance
     - plugins:
       - name: drf
-      - name: predicates
+      - name: devices
```

Comment:
In each doc, we should support two ways guidance, one is before v1.8.2, and another is after v1.8.2.